contracts/CCollateralCapErc20.sol

```diff
@@ -765,35 +767,17 @@ contract CCollateralCapErc20 is CToken, CCollateralCapErc20Interface {
         initializeAccountCollateralTokens(liquidator);
         initializeAccountCollateralTokens(borrower);
 
+        uint256 collateralTokens = getCollateralTokens(seizeTokens, borrower);
+
         /* Fail if seize not allowed */
         require(
-            comptroller.seizeAllowed(address(this), seizerToken, liquidator, borrower, seizeTokens) == 0,
+            comptroller.seizeAllowed(address(this), seizerToken, liquidator, borrower, collateralTokens) == 0,
```

Comment:
Passing `collateralTokens` into  comptroller seizeAllowed aligns with redeem and transfer behavior, although this parameter is not used in the hook.